exercism/largest-series-product/largest-series-product.js

```diff
@@ -0,0 +1,30 @@
+var Series = function(inputNumber) {
+  if( /[^0-9]/g.test(inputNumber) ) {
+    throw new Error('Invalid input.')
+  }
+
+  this.number = inputNumber
+};
+
+Series.prototype.largestProduct = function(seriesNumber) {
+  if( /[^0-9]/g.test(seriesNumber)) {
+    throw new Error('Invalid input.')
+  }
+  if (seriesNumber > this.number.length) {
+    throw new Error('Slice size is too big.')
+  }
+
+  let largestProduct = 0
+  for (let i = 0; i <= this.number.length-seriesNumber; i++) {
```

Comment:
can you place a space in between `this.number.length`, `seriesNumber`, and the `-`?